### ISSUE TITLE
Bad font rendering in dark mode.

### ISSUE THREAD

**Issue by stackingflow** [davidbailey00/notion-linux#50]
I have been using lots of electron applications but Notion-Linux has some font rendering issue especially in dark mode (white text on dark background). Please click on image or open in new window for better preview.

![notionlinuxxx](https://user-images.githubusercontent.com/47140829/118627773-a1845780-b7e9-11eb-8190-6c533ec9936b.png)

I have attached some screenshots of various application.

Discord: 
![notion-linux4](https://user-images.githubusercontent.com/47140829/118626723-bf9d8800-b7e8-11eb-854a-66b10e8c54fc.png)

Visual Studio Code:
![notion-linux1](https://user-images.githubusercontent.com/47140829/118626767-c926f000-b7e8-11eb-9d09-bd7a68a06e57.png)

Obsidian:
![notion-linux](https://user-images.githubusercontent.com/47140829/118626825-d217c180-b7e8-11eb-8be4-0f26e7f0bf0c.png)

Bitwarden also had a similar issue
https://github.com/bitwarden/desktop/issues/608
https://community.bitwarden.com/t/improve-font-rendering-in-desktop-app-by-setting-electron-background/16430
https://github.com/electron/electron/blob/master/docs/faq.md#the-font-looks-blurry-what-is-this-and-what-can-i-do
https://stackoverflow.com/questions/39748484/blurry-text-electron

**Comment by validatedev** [comment]
Are you using fractional scaling? Maybe that is the issue.

**Comment by stackingflow** [comment]
> Are you using fractional scaling? Maybe that is the issue.

Yes I had fractional scaling turned ON. I can't see any difference turning that OFF.

**Comment by vtenfys** [comment]
I'm afraid I am no longer supporting this project, so I will not be looking into this issue. Please see the updated README for more information. I will leave the issue open, in case of a new maintainer who is able to look into it.
